traffic-id.yaml

```diff
@@ -126,6 +157,84 @@ tls-sni-patterns:
       - instagram.com
       - cdninstagram.com
 
+  - id: feedly
+    labels:
+      - news
+      - rss
+    patterns:
+      - feedly.com
+
+  - id: nos
+    labels:
+      - news
+    patterns:
+      - nos.nl
+
+  - id: chartbeat
+    labels:
+      - stats
+      - tracking
+    patterns:
+      - chartbeat.net
+
+  - id: github
+    labels:
+      - dev
+    patterns:
+      - github.com
+
+  - id: github-tracking
+    labels:
+      - dev
+      - stats
+      - tracking
+    patterns:
+      - collector.githubapp.com
+
+  - id: mozilla
+    labels:
+      - browser
+    patterns:
+      - firefoxusercontent.com
+      - services.mozilla.com
+
+  - id: rememberthemilk
+    labels:
+      - todo
```

Comment:
perhaps here we could have something like 'pim' (personal information management) ?

Reply:
Maybe. But again, but this would only apply to a subset as well.  Asana for instance is more enterprisey todo/project management. Which remember the milk can do as well, so "personal" is misleading here.